pkg/cloudprovider/providers/openstack/openstack_volumes.go

```diff
@@ -74,8 +74,14 @@ type VolumeCreateOpts struct {
 	Metadata     map[string]string
 }
 
-func (volumes *VolumesV1) createVolume(opts VolumeCreateOpts) (string, error) {
+const (
+	volumeAvailableStatus = "available"
+	volumeInUseStatus = "in-use"
+	VolumeDeletedStatus = "deleted"
+	VolumeErrorStatus = "error"
+)
```

Comment:
Lets be consistent about naming. Constants generally start with upper case letter.